### DISCUSSION TITLE
why it has changed chinese ,i didn't set Configure Display Language，Even if I set it to en

### DISCUSSION THREAD

**Discussion by alex20230721** [microsoft/vscode-cmake-tools#4049] [Q&A]
<img width="955" alt="image" src="https://github.com/user-attachments/assets/e59e79aa-478c-4349-97b7-cbbf43b3f1cf">
Why is it in Chinese? 

The cmake bar on the left, 
run in the lower left corner, 
and the prompt message in the lower right corner all changed chinese ,and There are many more places

Did it automatically configure the plugin's language according to the system language ?
It wasn't like this a few days ago.

I need an English env, although I am Chinese,
pls tell me how to change english env

**Comment by gcampbell-msft** [comment]
@v-ericawu @v-frankwang Are you able to reproduce this?

@alex20230721 I have not been able to reproduce this. I see the same language as my display language on both the most recent official version and the latest pre-release version. Thanks!

**Comment by v-frankwang** [comment]
@gcampbell-msft @alex20230721 I can't reproduce the issue in either the latest production version or the latest pre-release version, and it won't show as Chinese unless I set the display language to Chinese.

Actual Result：
![4049](https://github.com/user-attachments/assets/78889dcf-710b-427f-9016-dc2a168dcb6d)

**Comment by alex20230721** [comment]
<img width="1004" alt="截屏2024-10-02 16 15 33" src="https://github.com/user-attachments/assets/ae84d603-ce7a-4f02-bb1b-38c4b71a6ecd">

**Comment by alex20230721** [comment]
it may be a problem with vscode，I'll go there and raise the issue.

**Reply by alex20230721** [reply]
the latest vscode and cmake

**Reply by gcampbell-msft** [reply]
@alex20230721 Are you saying you determined that this issue is with VSCode? 

We use and rely on the `vscode-nls` package for our localization. https://www.npmjs.com/package/vscode-nls

**Comment by alex20230721** [comment]
Do you need any logs from me? If so, please tell me how to find them.

**Comment by gcampbell-msft** [comment]
@alex20230721 You had mentioned that you were going to file an issue with VS Code, have you done that? 

Have you attempted to change the display language in VS Code and then restart VS Code?

**Comment by gcampbell-msft** [comment]
Based on our own investigation and inability to reproduce, I'm going to close this. However, please open up an issue, rather than a discussion post, if this continues to show for you.
